mta/mta_services.go

```diff
@@ -321,6 +330,23 @@ func UpdateBuildParameters(path string, buildParamsDataJSON string) ([]string, e
 	return messages, saveMTA(path, mta, Marshal)
 }
 
+//UpdateGlobalParameters - updates the MTA global parameters.
+func UpdateGlobalParameters(path string, globalParamsDataJSON string) ([]string, error) {
+	mta, messages, err := GetMtaFromFile(path, nil, false)
+	if err != nil {
+		return messages, err
+	}
+
+	globalParams := map[string]interface{}{}
```

Comment:
Did you run the command from the executable to check this doesn't fail? (Not sure if it will take the `string` key or require `interface{}`...)